src/haconiwa/scan/formatter.py

```
@@ -0,0 +1,253 @@
+        
+        return "\n".join(lines)
+    
+    def _format_json(self, data: Any) -> str:
+        """Format as JSON"""
+        return json.dumps(data, indent=2, default=str)
+    
+    def _format_yaml(self, data: Any) -> str:
+        """Format as YAML"""
+        return yaml.dump(data, default_flow_style=False, allow_unicode=True)
+    
+    def _format_summary(self, data: Any) -> str:
```

Pylint (3.3.7)

[refactor] Too many branches (16/12)

(R0912)